Tests/Parser/fixtures/oss.yml

```diff
@@ -2935,3 +2935,115 @@
     version: 9.4.0
     platform: ""
     family: GNU/Linux
+-
+  user_agent: com.apple.Safari.SearchHelper/14607.1.40.1.4 CFNetwork/978.0.7 Darwin/18.5.0 (x86_64)
+  os:
+    name: Mac
+    short_name: MAC
+    version: 10.14.4
+    platform: x64
+    family: Mac
+-
+  user_agent: com.apple.Safari.SearchHelper/14607.2.6.1.1 CFNetwork/978.0.7 Darwin/18.6.0 (x86_64)
+  os:
+    name: Mac
+    short_name: MAC
+    version: 10.14.5
+    platform: x64
+    family: Mac
+-
+  user_agent: com.apple.Safari.SearchHelper/12602.1.50.0.2 CFNetwork/807.0.4 Darwin/16.0.0 (x86_64)
+  os:
+    name: Mac
+    short_name: MAC
+    version: "10.12"
+    platform: x64
+    family: Mac
+-
+  user_agent: iTunes/12.9.5 (Macintosh; OS X 10.14.6) AppleWebKit/607.3.9
+  os:
+    name: Mac
+    short_name: MAC
+    version: 10.14.6
+    platform: ""
+    family: Mac
+-
+  user_agent: Safari/10600.1.25 CFNetwork/720.0.7 Darwin/14.0.0 (x86_64)
+  os:
+    name: Mac
+    short_name: MAC
+    version: "10.10"
+    platform: x64
+    family: Mac
+-
+  user_agent: Safari/12602.4.8 CFNetwork/807.2.14 Darwin/16.4.0 (x86_64)
+  os:
+    name: Mac
+    short_name: MAC
+    version: 10.12.3
+    platform: x64
+    family: Mac
+-
+  user_agent: Safari/6531.9 CFNetwork/454.4 Darwin/10.0.0 (i386) (MacBook4%2C1)
+  os:
+    name: Mac
+    short_name: MAC
+    version: "10.6"
+    platform: x86
+    family: Mac
+-
+  user_agent: Safari/8537.85.17.9.1 CFNetwork/596.6.4 Darwin/12.6.0 (x86_64) (iMac13%2C2)
+  os:
+    name: Mac
+    short_name: MAC
+    version: 10.8.5
+    platform: x64
+    family: Mac
+-
+  user_agent: Safari/6533.21.1 CFNetwork/454.11.12 Darwin/10.7.4 (x86_64) (MacBookPro8%2C1)
+  os:
+    name: Mac
+    short_name: MAC
+    version: 10.6.7
+    platform: x64
+    family: Mac
+-
+  user_agent: GoogleEarth/4.2.0205.5730(Macintosh;Mac OS X (10.5.4);en;kml:2.2;client:Free;type:default)
+  os:
+    name: Mac
+    short_name: MAC
+    version: 10.5.4
+    platform: ""
+    family: Mac
+-
+  user_agent: GoogleEarth/7.0.3.8542(Windows;Microsoft Windows (6.1.7601.1);pt;kml:2.2;client:Free;type:default)
+  os:
+    name: Windows
+    short_name: WIN
+    version: 6.1.7601.1
```

Comment:
version 7
all list 
Operating System | Version Number
-- | --
Windows 1.0 | 1.04
Windows 2.0 | 2.11
Windows 3.0 | 3
Windows NT 3.1 | 3.10.528
Windows for Workgroups 3.11 | 3.11
Windows NT Workstation 3.5 | 3.5.807
Windows NT Workstation 3.51 | 3.51.1057
Windows 95 | 4.0.950
Windows NT Workstation 4.0 | 4.0.1381
Windows 98 | 4.1.1998
Windows 98 Second Edition | 4.1.2222
Windows Me | 4.90.3000
Windows 2000 Professional | 5.0.2195
Windows XP | 5.1.2600
Windows Vista | 6.0.6000
Windows 7 | 6.1.7600
Windows 8.1 | 6.3.9600
Windows 10 | 10.0.10240